homeassistant/components/tcp.py

```diff
@@ -0,0 +1,125 @@
+"""
+homeassistant.components.tcp
```

Comment:
Instead of creating a generic entity, I think we're fine with having the binary sensor depend on the sensor. We do the same for REST.